modules/contributor-guide/partials/proc_testing-che-theia-with-custom-branding.adoc

```diff
@@ -23,7 +23,7 @@ To test a custom Che-Theia image, create a new `meta.yaml` file describing a cus
 
 . Create a workspace using the sample https://github.com/che-samples/che-theia-branding-example/blob/master/devfile.yaml[che-theia-branding-example devfile] to apply the changes:
 +
-image::https://www.eclipse.org/che/contribute.svg[link="https://che.openshift.io/#https://raw.githubusercontent.com/che-samples/che-theia-branding-example/master/devfile.yaml"]
+image::https://www.eclipse.org/che/contribute.svg[link="https://workspaces.openshift.com/#https://raw.githubusercontent.com/che-samples/che-theia-branding-example/master/devfile.yaml"]
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on f?url= see #1884